src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -38,6 +38,16 @@ const dataPathDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
   return settings;
 };
 
+const xsrfDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'server.xsrf.whitelist')) {
+    log(
+      'It is not recommended to disable xsrf protections for API endpoints via [server.xsrf.whitelist].' +
+        'It will be removed in 8.0 release. Instead, supply the "kbn-xsrf" header.'
```

Comment:
@joshdover what is the appropriate place to track v8 breaking changes after https://github.com/elastic/kibana/issues/40768 was closed?

Reply:
I believe this doc: `docs/migration/migrate_8_0.asciidoc`